gallery/lib/demo.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+class DemoPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Demo'),
+      ),
+      body: Center(
+        child: Text('Here is a demo!'),
+      ),
+    );
+  }
+}
```

Comment:
Be consistent about newlines at EOF (either always do or always don't)

Reply:
Done